omega/omega-transaction/src/test/java/org/apache/servicecomb/pack/omega/transaction/TransactionAspectTest.java

```diff
@@ -17,47 +17,43 @@
 
 package org.apache.servicecomb.pack.omega.transaction;
 
```

Comment:
Please use the google-style which could find how to setup the developement environment [here](http://servicecomb.apache.org/developers/setup-develop-environment/)